pkg/psmdb/init.go

```diff
@@ -0,0 +1,19 @@
+package psmdb
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+func EntrypointInitContainer(initImageName string) corev1.Container {
+	return corev1.Container{
+		VolumeMounts: []corev1.VolumeMount{
+			{
+				Name:      MongodDataVolClaimName,
+				MountPath: "/data/db",
+			},
+		},
+		Image:   initImageName,
+		Name:    "pxc-init",
+		Command: []string{"/init-entrypoint.sh"},
```

Comment:
init container should have the same resources requests/limits as the main container.
Could you please add?

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/362/commits/aab0abad6fa05278fd878a71c862cd133e5b1f33#diff-21cee571a261da42cae4cee0bf0952c6R78